### PR TITLE
[docs][yba][DOC-69] YCQL table restore

### PR DESCRIPTION
**Pull request by ddhodge**
Restore selected tables for YCQL backups

@netlify /preview/yugabyte-platform/back-up-restore-universes/

### CODE REVIEW

**Review by aishwarya24 — COMMENTED** [review]
will look into it more tomorrow.

**Review by haikarthikssk — APPROVED** [review]

File: docs/content/preview/yugabyte-platform/back-up-restore-universes/back-up-universe-data.md [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+
+1. Select the storage configuration you want to use for the backup. For more information, see [Configure backup storage](../configure-backup-storage/).
+
+1. Select the database (YSQL) or keyspace (YCQL) to back up.
+
+1. For YCQL backups, you can choose to back up all tables in the keyspace to which the database belongs or only certain tables. Click **Select a subset of tables** to display the **Select Tables** dialog, where you can select one or more tables to back up. Click **Confirm** when you are done.
+
+1. Specify the period of time during which the backup is to be retained. Note that there's an option to never delete the backup.
+
+1. Optionally, specify the number of threads that should be available for the backup process.
```
Comment (aishwarya24) [inline docs/content/preview/yugabyte-platform/back-up-restore-universes/back-up-universe-data.md:47]: The UI was not showing me this option . Need to check if this step can be removed. ( specify the number of threads that should be available for the backup process.)

File: docs/content/preview/yugabyte-platform/back-up-restore-universes/back-up-universe-data.md [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+
+Note that non-transactional backups are not supported.
+
+To view, [restore](../restore-universe-data/), or delete existing backups for your universe, navigate to that universe and select **Backups**.
+
+By default, the list displays all the backups generated for the universe regardless of the time period. You can configure the list to only display the backups created during a specific time period, such as last year, last month, and so on. In addition, you can specify a custom time period.
+
+## Create backups
+
+The **Backups** page allows you to create new backups that start immediately, as follows:
```
Comment (aishwarya24) [inline docs/content/preview/yugabyte-platform/back-up-restore-universes/back-up-universe-data.md:31]: This statement is confusing because there are two places where I see backups. One for a single universe and the other is for all universes in YBA. So I think , you can rephrase it to something like .. The Backups page for a universe allows you to create new backups that start immediately, as follows:

Comment (aishwarya24) [reply docs/content/preview/yugabyte-platform/back-up-restore-universes/back-up-universe-data.md:31]: I would also include this as the first bullet - 1. Navigate to a universe and select the Backups tab.

File: docs/content/preview/yugabyte-platform/back-up-restore-universes/schedule-data-backups.md [hunk 3]
```diff
@@ -51,11 +32,13 @@ Before scheduling a backup of your universe YCQL data, create a policy, as follo
...
-1. Ensure that the API type is set as YCQL.
+1. Select the API type for the backup.
+
+1. Select the backup storage configuration. The available configurations depend on your existing backup storage configurations. For more information, see [Configure backup storage](../configure-backup-storage/).
 
-1. Select the database to back up.
+1. Select the database/keyspace to back up.
 
-1. Specify whether you want to back up all tables in the keyspace to which the database belongs or only certain tables. If you choose **Select a subset of tables**, a **Select Tables** dialog opens allowing you to select one or more tables to back up.
+1. For YCQL backups, you can choose to back up all tables in the keyspace to which the database belongs or only certain tables. If you choose **Select a subset of tables**, a **Select Tables** dialog opens allowing you to select one or more tables to back up. When finished, click **Confirm**.
```
Comment (aishwarya24) [inline docs/content/preview/yugabyte-platform/back-up-restore-universes/schedule-data-backups.md:41]: On line 57 , there is a mention of parallel threads under bullet 11 which is not showing up in the UI.

**Review by aishwarya24 — APPROVED** [review]
LGTM with few comments.
